configure.ac

```diff
@@ -220,7 +220,7 @@ do
 done
 for i in templates cfe_internal
 do
-    for j in `find "$srcdir/$i" -name '*.mustache' -o -name '*.sh'`
+    for j in `find "$srcdir/$i" -name '*.mustache' -o -name '*.sh' -o -name '*.awk' -o -name '*.sed'`
```

Comment:
If we are making this nice change shouldn't we rename templates/federated_reporting/filter.sed.mustache to filter.sed?

Reply:
Yes, and we should also prefix with a number for ordering. Will do.